src/bindings/PyDP/base/logging.cpp

```diff
@@ -18,9 +18,7 @@ class Logging_helper {
   }
   int get_vlog_level() { return dpbase::get_vlog_level(); }
 
-  std::__cxx11::string get_log_directory() {
-    return dpbase::get_log_directory();
-  }
+  std::__cxx11::string get_log_directory() { return dpbase::get_log_directory(); }
```

Comment:
Do we want the function definition in the same-line?

Reply:
This is consistent with the Differential Privacy codebase (that is, when reading Google code, we will find this formatting). Once again, I don't have any strong argument against it (besides personal preference).

Reply:
Now clang does not format short functions in one line